lib/ipaddr.h

```diff
@@ -166,6 +166,12 @@ static inline bool ipaddr_is_zero(const struct ipaddr *ip)
 	return true;
 }
 
+static inline bool ipaddr_is_same(const struct ipaddr *ip1,
+				  const struct ipaddr *ip2)
+{
+	return (ipaddr_cmp(ip1, ip2) == 0);
```

Comment:
we don't put paranthesis on return statements please.

Reply:
Done